.pre-commit-config.yaml

```diff
@@ -67,6 +51,12 @@ repos:
         # https://github.com/rust-lang/rustfmt/issues/4485
         entry: rustfmt --edition 2021
         files: ^sentry_streams/.*\.rs$
+      - id: mypy
+        name: mypy
+        entry: mypy
+        files: ^sentry_streams/.*\.py$
+        types: [python]
+        language: system
```

Comment:
run mypy in surrounding venv so that it picks up installed dependnecies properly. this is also how we run it in sentry (although @asottile-sentry at some point said he doesn't want to use `language: system` that often in the future IIRC)